docs/ibc/light-clients/wasm/overview.md

```diff
@@ -0,0 +1,28 @@
+<!--
+order: 1
+-->
+
+# `08-wasm`
+
+## Overview
+
+Learn about the `08-wasm` light client proxy module. {synopsis}
+
+### Context
+
+Traditionally, light clients used by ibc-go have been implemented in Go, and since ibc-go v7 (with the release of the 02-client refactor), they are [first-class Cosmos SDK modules](../../../architecture/adr-010-light-clients-as-sdk-modules.md). This means that upgrading existing light client implementations or adding support for new light clients is a multi-step, time-consuming process involving on-chain governance: it is necessary to modify the codebase of ibc-go (if the light client is part of its codebase), re-build chains' binaries, pass a governance proposal and validators upgrade their nodes.
```

Comment:
```suggestion
Traditionally, light clients used by ibc-go have been implemented in Go, and since ibc-go v7 (with the release of the 02-client refactor), they are [first-class Cosmos SDK modules](../../../architecture/adr-010-light-clients-as-sdk-modules.md). This means that upgrading existing light client implementations or adding support for new light clients is a multi-step, time-consuming process involving on-chain governance: it is necessary to modify the codebase of ibc-go (if the light client is part of its codebase), re-build chains' binaries, pass a governance proposal and have validators upgrade their nodes.
```